File: src/test/java/kitchenpos/fixtures/MenuProductFixture.java
package kitchenpos.fixtures;

import kitchenpos.domain.MenuProduct;
import kitchenpos.domain.Product;

import java.util.UUID;

public class MenuProductFixture {

    public static long seq = 1L;

    public static MenuProduct create(Product product, long quantity, UUID productId) {
        MenuProduct menuProduct = new MenuProduct();
        menuProduct.setSeq(seq++);
        menuProduct.setProduct(product);
        menuProduct.setQuantity(quantity);
        menuProduct.setProductId(productId);
        return menuProduct;
    }
}

File: src/test/kotlin/kitchenpos/menu/MenuProductFixture.kt
package kitchenpos.menu

import kitchenpos.domain.MenuProduct
import kitchenpos.domain.Product

class MenuProductFixture {
    companion object {
        private const val DEFAULT_QUANTITY: Long = 1

        fun fixture(
            seq: Long,
            product: Product,
            quantity: Long = DEFAULT_QUANTITY,
        ): MenuProduct {
            val menuProduct = MenuProduct()
            menuProduct.seq = seq
            menuProduct.productId = product.id
            menuProduct.quantity = quantity
            return menuProduct
        }
    }
}
